packaging/helm/skipper-aws/README.md

```diff
@@ -0,0 +1,25 @@
+# Helm chart
+
+We expect AWS IAM roles with names "-role" names are setup for:
+
+- kube2iam: kube2iam-role
+- kube-aws-iam-controller: kube-aws-iam-controller-role
+- kube-ingress-aws-controller: kube-ingress-aws-controller-role
+- external-dns: external-dns-role
+
+See values.yaml to change installation options.
+
+If you want to setup skipper, kube-ingress-aws-controller,
+external-dns and kube2iam via helm in your test environment:
+
+```
+git clone https://github.com/zalando/skipper.git
+cd skipper/packaging/helm
+kubectl create -f skipper-aws/manifests/
```

Comment:
What's the reasoning for having these outside of Helm?

Reply:
Reasoning was, that these don't contain template code

Reply:
It also makes it easier to upgrade and uninstall everything. (e.g. `helm uninstall skipper` will remove everything that was installed with helm)

Reply:
good point, fixed